docs/src/main/sphinx/connector/iceberg.rst

```diff
@@ -1032,3 +1043,39 @@ like a normal view, and the data is queried directly from the base tables.
 
 Dropping a materialized view with :doc:`/sql/drop-materialized-view` removes
 the definition and the storage table.
+
+Table statistics
+----------------
+
+There is experimental support to collect column statistics which can be enabled by
+setting the ``iceberg.experimental.extended-statistics.enabled`` catalog
+configuration property or the corresponding
+``experimental_extended_statistics_enabled`` session property to ``true``.
+Enabling this configuration will allow executing :doc:`/sql/analyze` statement.
+
+.. _iceberg_analyze:
+
+Updating table statistics
+^^^^^^^^^^^^^^^^^^^^^^^^^
+
+If your queries are complex and include joining large data sets,
+running :doc:`/sql/analyze` on tables may improve query performance
```

Comment:
why is wrapping narrower here?

Reply:
What do you mean?

Reply:
Are we following line wrapping limitation strictly? I think no.